client/src/elements/item-preview/preview-container.css

```diff
@@ -1,26 +1,8 @@
 preview-container {
   position: relative;
   display: block;
-  margin-top: calc(var(--q-space-base) + 9px);
-  margin-bottom: calc(var(--q-space-base) + 9px);
-
-  padding: var(--q-space-base) calc(var(--q-space-base) * 2);
-}
-
-preview-container::before, preview-container::after {
-  content: '';
-  display: block;
-  height: 9px;
+  height: 350px;
```

Comment:
does it make sense to limit the height to 350px?
If there are a lot of options we have a lot of space to display potentially much higher graphics.

Could we set this to `min-height: 350px;` and `height: 100%;` and then use flexbox in the `editor.html` to let the preview section expand to the same height as the options need?

in `item-overview` the height could be limited to `350px` on the `item-preview` element.

See this bin for an example: http://jsbin.com/vuzodozula/edit?html,css,output

this is not completely thought through, but you probably get the idea.